### PR TITLE
Docs for Application Default Credentials, links to non-dwdoa setup

### PR DESCRIPTION
**Pull request by torfjor**

### CODE REVIEW

File: website/content/docs/auth/jwt/oidc-providers/google.mdx [hunk 1]
```diff
@@ -30,19 +30,20 @@ used to copy claims data into resulting auth token and alias metadata via [claim
...
-Google Workspace Domain-Wide Delegation of Authority for authentication and authorization. You need to follow
-**all steps** in the [guide](https://developers.google.com/workspace/guides/create-credentials#service-account)
-to obtain the key file for a Google service account capable of making requests to the Google Workspace
-[User Accounts](https://developers.google.com/admin-sdk/directory/v1/guides/manage-users) and
-[Groups](https://developers.google.com/admin-sdk/directory/v1/guides/manage-groups) APIs.
+The Google-specific handling that's used to fetch Google Workspace groups and user information in Vault uses either
+Google Workspace Domain-Wide Delegation of Authority for authentication and authorization, or group admin roles granted to a GCP service account.
+
+Links to steps for setting up authentication and authorization:
+- [DWDoA](https://developers.google.com/workspace/guides/create-credentials#service-account)
```
Comment (austingebauer) [inline website/content/docs/auth/jwt/oidc-providers/google.mdx:45]:
```suggestion
- [With Domain-Wide Delegation of Authority](https://developers.google.com/workspace/guides/create-credentials#service-account)
```

File: website/content/docs/auth/jwt/oidc-providers/google.mdx [hunk 2]
```diff
@@ -30,19 +30,20 @@ used to copy claims data into resulting auth token and alias metadata via [claim
...
+- [Without DWDoA](https://cloud.google.com/identity/docs/how-to/setup#auth-no-dwd)
```
Comment (austingebauer) [inline website/content/docs/auth/jwt/oidc-providers/google.mdx:46]:
```suggestion
- [Without Domain-Wide Delegation of Authority](https://cloud.google.com/identity/docs/how-to/setup#auth-no-dwd)
```

File: website/content/docs/auth/jwt/oidc-providers/google.mdx [hunk 3]
```diff
@@ -57,15 +58,20 @@ that enable the feature.
...
 - `fetch_user_info` `(bool: false)` - If set to true, user info will be fetched from Google Workspace using the configured [user_custom_schemas](#user_custom_schemas).
 - `groups_recurse_max_depth` `(int: <optional>)` - Group membership recursion max depth. Defaults to 0, which means don't recurse.
 - `user_custom_schemas` `(string: <optional>)` - Comma-separated list of Google Workspace [custom schemas](https://developers.google.com/admin-sdk/directory/v1/guides/manage-schemas).
   Values set for Google Workspace users using custom schema fields will be fetched and made available as claims that can be used with [claim_mappings](/vault/api-docs/auth/jwt#claim_mappings). Required if [fetch_user_info](#fetch_user_info) is set to true.
+- `impersonate_principal` `(string: <optional>)` - Service account email that has been granted domain-wide delegation of authority in Google Workspace.
+  Required if accessing the Google Workspace Directory API through domain-wide delegation of authority, without using a service account key.
+  The service account vault is running under must be granted the `iam.serviceAccounts.signJwt` permission on this service account.
```
Comment (austingebauer) [inline website/content/docs/auth/jwt/oidc-providers/google.mdx:73]:
```suggestion
  The service account Vault is running under must be granted the `iam.serviceAccounts.signJwt` permission on this service account.
```

File: website/content/docs/auth/jwt/oidc-providers/google.mdx [hunk 4]
```diff
@@ -30,19 +30,20 @@ used to copy claims data into resulting auth token and alias metadata via [claim
...
-[User Accounts](https://developers.google.com/admin-sdk/directory/v1/guides/manage-users) and
-[Groups](https://developers.google.com/admin-sdk/directory/v1/guides/manage-groups) APIs.
+The Google-specific handling that's used to fetch Google Workspace groups and user information in Vault uses either
+Google Workspace Domain-Wide Delegation of Authority for authentication and authorization, or group admin roles granted to a GCP service account.
+
+Links to steps for setting up authentication and authorization:
+- [DWDoA](https://developers.google.com/workspace/guides/create-credentials#service-account)
+- [Without DWDoA](https://cloud.google.com/identity/docs/how-to/setup#auth-no-dwd)
 
 In **step 11** within the section titled
```
Comment (austingebauer) [inline website/content/docs/auth/jwt/oidc-providers/google.mdx:48]:
```suggestion
In **step 5e** within the section titled
```

**Review by austingebauer — COMMENTED** [review]
Looking good. Left a few suggestions. Thanks for opening this!

File: website/content/docs/auth/jwt/oidc-providers/google.mdx [hunk 5]
```diff
@@ -30,19 +30,20 @@ used to copy claims data into resulting auth token and alias metadata via [claim
...
   Google account to authenticate with Vault.
 
-The Google-specific handling that's used to fetch Google Workspace groups and user information in Vault uses
-Google Workspace Domain-Wide Delegation of Authority for authentication and authorization. You need to follow
-**all steps** in the [guide](https://developers.google.com/workspace/guides/create-credentials#service-account)
-to obtain the key file for a Google service account capable of making requests to the Google Workspace
-[User Accounts](https://developers.google.com/admin-sdk/directory/v1/guides/manage-users) and
-[Groups](https://developers.google.com/admin-sdk/directory/v1/guides/manage-groups) APIs.
+The Google-specific handling that's used to fetch Google Workspace groups and user information in Vault uses either
+Google Workspace Domain-Wide Delegation of Authority for authentication and authorization, or group admin roles granted to a GCP service account.
```
Comment (schavis) [inline website/content/docs/auth/jwt/oidc-providers/google.mdx:42]:
```suggestion
Vault uses one of the following methods to fetch Google Workspace groups and user information:

- Google Workspace Domain-Wide Delegation of Authority for authentication and authorization
- Group admin roles granted to a GCP service account.
```

Suggest simplifying to make the text easier to scan and parse.

**Review by schavis — CHANGES_REQUESTED** [review]

**Comment by austingebauer** [comment]
@torfjor - Curious if you'd be able to apply some of the suggestions here? If not, I'm happy to merge this PR and take over with the final edits. Thanks!

**Comment by austingebauer** [comment]
@torfjor - I'm going to go ahead and merge this. I'll fix up the docs myself for the upcoming release.

**Review by austingebauer — APPROVED** [review]

**Pull request merged**